src/renderers/webgl/WebGLPrograms.js

```diff
@@ -208,7 +208,7 @@ function WebGLPrograms( renderer, cubemaps, extensions, capabilities, bindingSta
 
 			vertexTangents: ( material.normalMap && material.vertexTangents ),
 			vertexColors: material.vertexColors,
-			vertexAlpha: ( ( object.isMesh || object.isLine || object.isPoints ) && object.geometry.attributes.color && object.geometry.attributes.color.itemSize === 4 ),
+			vertexAlphas: material.vertexColors && object.geometry.attributes.color && object.geometry.attributes.color.itemSize === 4,
```

Comment:
We are not deprecating `material.vertexColors` yet so the right way to do it to me.

Reply:
This looks good!